daemon/exec.go

```diff
@@ -276,15 +277,15 @@ func (daemon *Daemon) ContainerExecStart(ctx context.Context, name string, stdin
 	select {
 	case <-ctx.Done():
 		logrus.Debugf("Sending TERM signal to process %v in container %v", name, c.ID)
-		daemon.containerd.SignalProcess(ctx, c.ID, name, int(signal.SignalMap["TERM"]))
+		daemon.ContainerExecKill(ctx, name, uint64(signal.SignalMap["TERM"]))
```

Comment:
Given that we already have the exec config, it seems better to call the contaunerd API directly than to grab this info again.